notebooks_community/vbll_thompson_sampling.ipynb

```diff
@@ -0,0 +1,803 @@
+      "text/plain": [
+       "<Figure size 600x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def plot_model(\n",
+    "    model,\n",
+    "    x,\n",
+    "    y,\n",
+    "    new_data=None,\n",
+    "    show_opjective=False,\n",
+    "    title=\"VBLL Predictive on Toy Data with Thompson Samples\",\n",
+    "):\n",
+    "    x_test = torch.linspace(0, 1, 100)\n",
+    "    with torch.no_grad():\n",
+    "        posterior = model.posterior(x_test.view(-1, 1))\n",
+    "    mean, std = posterior.mean.squeeze(), posterior.variance.sqrt().squeeze()\n",
+    "\n",
+    "    fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(6, 4))\n",
+    "    ax.set_title(title)\n",
+    "    ax.scatter(x, y, c=\"k\", label=\"Observations\")\n",
+    "\n",
+    "    if new_data is not None:\n",
+    "        x_new, y_new = new_data\n",
+    "        ax.scatter(x_new, y_new, c=\"r\", label=\"New observations\")\n",
+    "\n",
+    "    if show_opjective:\n",
+    "        ax.plot(x_test, objective(x_test, noise=False), label=\"Objective\", color=\"k\")\n",
+    "\n",
+    "    ax.plot(x_test, mean, label=\"Posterior predictive\", color=\"tab:blue\")\n",
+    "\n",
+    "    # Posterior samples\n",
```

Comment:
The TS samples shown are not the samples that were optimized to obtain the next candidate point. This initially caused some confusion; can you make the plots in a way so that this is consistent (i.e. that we show the set of TSs that were optimized to obtain the next candidate)?